### ISSUE TITLE
Create home screen widget via long press menu

### ISSUE THREAD

**Issue by mueller-ma** [openhab/openhab-android#1535]
... like NFC item update tags are written.

**Comment by Kshitij09** [comment]
Can you explain a bit more ? I can work on this one.

**Comment by mueller-ma** [comment]
If you have a smart phone with NFC, you can long press nearly any widget on the sitemap (the main app screen) and select "Write item update tag" and select the state afterwards. This will write a NFC tag that changes the selected item to the selected state.
Currently it's possible to create home screen widgets from the widgets picker in your launcher, but IMO it would be handy if it's also possible via the same long press menu than NFC tags.

**Comment by mueller-ma** [comment]
I'm working on this.

**Comment by schuellerf** [comment]
It does not work in my case. The Widget just shows the icon and a ":" also touching the icon does not result in an action. I tested version "2.10.13-beta 21.11.2019 16:10:46"
I tried both a switch and a roller shutter.
<img src="https://user-images.githubusercontent.com/8057963/69476314-d2442600-0dd8-11ea-823b-57ac22ac2e40.jpg" width=40% /> <img src="https://user-images.githubusercontent.com/8057963/69476310-c5273700-0dd8-11ea-827b-9157d18e8d04.jpg" width=40% />

Should we create another ticket for that or re-open this one?
